tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -1924,6 +1924,34 @@ def test_filter_not_unresolved(self):
         assert response.status_code == 200
         assert [int(r["id"]) for r in response.data] == [event.group.id]
 
+    def test_default_search(self):
```

Comment:
could we add a resolved issue here and make sure that it doesn't show up in the default? or is there already one set up by the test harness